File: go-sdk/pkg/models/operations/listorganizations.go
package operations

type ListOrganizationsQueryParams struct {
	Page    *float64 `queryParam:"style=form,explode=true,name=page"`
	PerPage *float64 `queryParam:"style=form,explode=true,name=per_page"`
}

type ListOrganizationsRequest struct {
	QueryParams ListOrganizationsQueryParams
}

type ListOrganizationsResponse struct {
	ContentType                               string
	StatusCode                                int64
	ListOrganizations200ApplicationJSONObject map[string]map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/listorganizations.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class ListOrganizationsQueryParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "queryParam, style=form;explode=true;name=page" })
  page?: number;

  @SpeakeasyMetadata({ data: "queryParam, style=form;explode=true;name=per_page" })
  perPage?: number;
}

export class ListOrganizationsRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  queryParams: ListOrganizationsQueryParams;
}

export class ListOrganizationsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  listOrganizations200ApplicationJSONObject?: Record<string, Record<string, any>>;
}